symengine/llvm_double.cpp

```diff
@@ -495,7 +495,9 @@ llvm::Function *LLVMVisitor::get_powi()
 {
     std::vector<llvm::Type *> arg_type;
     arg_type.push_back(get_float_type(&mod->getContext()));
+#if (LLVM_VERSION_MAJOR > 12)
```

Comment:
I am not sure I understand this. So for older versions of LLVM only one `arg_type` will be used, but two for versions 13 and forward.

Reply:
that's right - for older LLVM the second arg is implicitly `int32`, and setting it explicitly appears to be harmless but gives the above warning - while for recent LLVM it needs to be set explicitly otherwise it will segfault (see #1810 for more details)

Reply:
Ok, understood